e2e/_suites/fleet/fleet.go

```diff
@@ -175,7 +175,7 @@ func (fts *FleetTestSuite) anStaleAgentIsDeployedToFleetWithInstaller(image, ver
 	return fts.anAgentIsDeployedToFleetWithInstaller(image, installerType)
 }
 
-func (fts *FleetTestSuite) installCerts(targetOS string) error {
```

Comment:
Input parameter not used internally. This targetOS is known by the installer that installed the agent